kotlinsample/src/main/java/com/airbnb/epoxy/kotlinsample/models/ItemTitleTextView.kt

```diff
@@ -0,0 +1,30 @@
+package com.airbnb.epoxy.kotlinsample.models
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.ViewGroup
+import androidx.appcompat.widget.AppCompatTextView
+import com.airbnb.epoxy.ModelView
+import com.airbnb.epoxy.TextProp
+
+@ModelView(autoLayout = ModelView.Size.MANUAL)
+class ItemTitleTextView @JvmOverloads constructor(
```

Comment:
Name this for what it is meant to be an example for 
```suggestion
class ManualLayoutParamsView @JvmOverloads constructor(
```